__device-tests__/utils.js

```diff
@@ -157,3 +157,98 @@ export async function takeScreenshot( { withoutKeyboard, crop } = {} ) {
 	const resizedImage = await image.resize( widthSize, jimp.AUTO );
 	return resizedImage.getBufferAsync( jimp.MIME_PNG );
 }
+
+/**
+ * Generates the appropriate URL to fetch the theme data based on the provided theme details.
+ *
+ * @param {Object}  theme                  Object containing details about the theme.
+ * @param {string}  theme.name             The name of the theme.
+ * @param {boolean} theme.isWordPressTheme Flag indicating whether the theme is a WordPress theme. Defaults to false.
```

Comment:
Nitpick: If this parameter is optional we can wrap it with brackets.
```suggestion
 * @param {boolean} [theme.isWordPressTheme] Flag indicating whether the theme is a WordPress theme. Defaults to false.
```

Reply:
Updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6187/commits/976e642d8648890d35522140348080460a5dd706 thanks for the suggestion!